### ISSUE TITLE
Map - dark night and strobe effect

### ISSUE THREAD

**Issue by Cuel** [acemod/ACE3#6298]
**Arma 3 Version:** 1.82.144647
**CBA Version:**  3.6.1.180131
**ACE3 Version:** 3.12.2.33

**Mods:**
```
- CBA_A3
- ace
```

**Description:**
- Looking at the map when near a light causes a strobe effect

**Steps to reproduce:**
- Look at the map when moving around the camp light

**Where did the issue occur?**
- SP/MP

**Repo file**
[mission.zip](https://github.com/acemod/ACE3/files/1939985/mission.zip)

**Comment by PabstMirror** [comment]
Possibly from https://github.com/acemod/ACE3/pull/6087 ?

**Comment by dedmen** [comment]
Had same blinking issue with only sunlight.
>chernarus redux. 093131 06.06.2030 19:40-19:50 the map is flickering like crazy. Aka the ace map darkness.
Time aceleration is set to... about 2-4x I think
flickering is gone at 19:51. Others have it too

**Comment by Cuel** [comment]
Yeah the reports I've had from my community is that it happens with several light sources not just camping lights
